site/content/blogs/release-v118.en.md

```diff
@@ -22,8 +22,69 @@ enabling you to focus on developing your application, instead of writing deploym
 See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
 ## Certificate import
+
 _Contributed by [Penghao He](https://github.com/iamhopaul123/)_
 
+Developers that have domains managed outside of Route 53 or who want to use HTTPS for Copilot applications without any domain specified can now use a new `--import-cert-arns` flag to import any validated certificates when they create an environment.
+
+```
+$ copilot env init --import-cert-arns arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+```
+
+For example, the certificate has `example.com` as its domain and `*.example.com` as a subject alternative name (SAN):
+
+```
+$ aws acm describe-certificate --certificate-arn arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+
+{
+  "Certificate": {
+    "CertificateArn": "arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012",
+    "DomainName": "example.com",
+    "SubjectAlternativeNames": [
+      "*.example.com"
+    ],
+    "DomainValidationOptions": [
+      {
+        "DomainName": "example.com",
+        "ValidationDomain": "example.com",
+        "ValidationStatus": "SUCCESS",
+        "ResourceRecord": {
+          "Name": "_45c8aa9ac85568e905a6c3852e62ebc6.example.com.",
+          "Type": "CNAME",
+          "Value": "_f8be688050b7d23184863690b3d4baa8.xrchbtpdjs.acm-validations.aws."
+        },
+        "ValidationMethod": "DNS"
+      }
+    ],
+    ...
+}
+```
+
+Then, aliases that are valid against any of the imported certificates can be specified in a [Load Balanced Web Service manifest](../manifest/lb-web-service.en.md) to deploy a service to the environment:
```

Comment:
```suggestion
Then, you can specify aliases that are valid against any of the imported certificates in a [Load Balanced Web Service's manifest](../manifest/lb-web-service.en.md):
```

Also, since this sentence mentions `"any of the imported certificates"`, do you think it'd flow better if the example above import two certificates?